combat-trainer.lic

```diff
@@ -1181,8 +1184,14 @@ class SpellProcess
     return unless DRStats.necromancer?
     return unless @necromancer_healing
     return if game_state.casting
-    return unless game_state.prepare_consume
 
+    if health <= @siphon_vit_threshold && @necromancer_healing['Siphon Vitality'] && DRRoom.npcs.any?
```

Comment:
Should `DRRoom.npcs` here be `game_state.npcs`? The latter will filter out anything in `settings.ignored_npcs` (which I _think _ would include a zombie or construct).

Reply:
Yup, good call. Thanks!

Reply:
Done.